gum/backend-windows/gummemoryaccessmonitor-windows.c

```diff
@@ -242,14 +268,101 @@ gum_set_guard_flag (const GumLiveRangeDetails * details,
                     gpointer user_data)
 {
   GumMemoryAccessMonitor * self = GUM_MEMORY_ACCESS_MONITOR (user_data);
-  DWORD old_prot;
+  GumMemoryAccessMonitorPrivate * priv = self->priv;
+  DWORD old_prot, new_prot;
   BOOL success;
+  gboolean is_guarded = FALSE;
+  guint num_pages;
+  /* dummy init to make VS happy... */
+  new_prot = PAGE_NOACCESS;
+
+  if (priv->access_mask & GUM_PAGE_READ)
```

Comment:
For consistency this should be:

`  if ((priv->access_mask & GUM_PAGE_READ) != 0)`

The rest of the code tries to be very explicit, so unless it's a gboolean we check explicitly against a value.